CrackingTheCodingInterview/Chapter1ArraysAndStrings/Permutation.java

```diff
@@ -1,7 +1,13 @@
-// Given two strings, write a metho dto decide if one is a permutation of the other
+//Different examples of implementations of arrays and strings to determine permutations. 
 
 public class Permutation {
-	public boolean permutation(String s, String t) {
+	/**
+	*boolean method to compare two Strings and determine whether they are permutations of each other.
+	*@param two Strings
```

Comment:
Wrong javadoc: http://www.oracle.com/technetwork/articles/java/index-137868.html.

Need an `@param` for each parameter:
    @param s The first string
    @param t The second string